Tribler/Main/vwxGUI/channel.py

```diff
@@ -1001,29 +1001,6 @@ def startDownload(self, torrentfilename, *args, **kwargs):
             print_exc()
         return False
 
-    def startDownloads(self, filenames, *args, **kwargs):
```

Comment:
There are two functions, one is that `startDownloads`, the other is `startDownload` (no "s" at the end). Only the later one is used.